### PR TITLE
Add JP60 keyboard

### PR DESCRIPTION
**Pull request by ai03-2725**
## Description

Adds support for the [JP60](https://github.com/ai03-2725/JP60) open-source JIS keyboard kit.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Adds support for JP60

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ai03/jp60/readme.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+GH60-compatible JIS-like hotswap keyboard PCB
+
+* Keyboard Maintainer: [ai03](https://github.com/ai03-2725)
+* Hardware Supported: [JP60 PCB](https://github.com/ai03-2725/JP60)
+* Hardware Availability: To be made available in JP market soon; will be listed in the repo above
+
+Make example for this keyboard (after setting up your build environment):
+
+    make jp60:default
```
Comment (zvecr) [inline keyboards/ai03/jp60/readme.md:13]:
```suggestion
    make ai03/jp60:default
```

File: keyboards/ai03/jp60/readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Flashing example for this keyboard:
+
+    make jp60:default:flash
```
Comment (zvecr) [inline keyboards/ai03/jp60/readme.md:17]:
```suggestion
    make ai03/jp60:default:flash
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/ai03/jp60/readme.md (hunk 1 above)
Comment (ai03-2725) [reply keyboards/ai03/jp60/readme.md]: Whoops. No clue how I missed this.
Fixed with [e558964](https://github.com/qmk/qmk_firmware/pull/14219/commits/e558964fd92bbf655aabe4414929ed96d569a4bb).

File: keyboards/ai03/jp60/readme.md (hunk 2 above)
Comment (ai03-2725) [reply keyboards/ai03/jp60/readme.md]: Resolved with [e558964](https://github.com/qmk/qmk_firmware/pull/14219/commits/e558964fd92bbf655aabe4414929ed96d569a4bb) as well.

File: keyboards/ai03/jp60/keymaps/default/config.h [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* Increase polling rates and scan quantity for improved performance */
+#define USB_POLLING_INTERVAL_MS 1
+#define QMK_KEYS_PER_SCAN 12
```
Comment (drashna) [inline keyboards/ai03/jp60/keymaps/default/config.h:21]:
```suggestion
#define QMK_KEYS_PER_SCAN 12

```

File: keyboards/ai03/jp60/keymaps/via/config.h [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* Increase polling rates and scan quantity for improved performance */
+#define USB_POLLING_INTERVAL_MS 1
+#define QMK_KEYS_PER_SCAN 12
```
Comment (drashna) [inline keyboards/ai03/jp60/keymaps/via/config.h:21]:
```suggestion
#define QMK_KEYS_PER_SCAN 12

```

File: keyboards/ai03/jp60/rules.mk [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
```
Comment (drashna) [inline keyboards/ai03/jp60/rules.mk:10]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Comment by ai03-2725** [comment]
Wanted to follow up just in case:
Is there something I still need to do for having the PR approved?

File: keyboards/ai03/jp60/info.json [hunk 6]
```diff
@@ -0,0 +1,344 @@
+{
+    "keyboard_name": "JP60",
+    "url": "https://github.com/ai03-2725/JP60",
+    "maintainer": "ai03",
+    "width": 15,
+    "height": 5,
```
Comment (drashna) [inline keyboards/ai03/jp60/info.json:6]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by drashna** [comment]
> Wanted to follow up just in case: Is there something I still need to do for having the PR approved?

Mostly, just needs a second review approval.

File: keyboards/ai03/jp60/rules.mk [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (fauxpark) [inline keyboards/ai03/jp60/rules.mk:21]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]
Thanks!

**Pull request merged**